common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -492,4 +492,14 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = false,
   )
+
+  val NewsletterSignupLayout = Switch(
+    SwitchGroup.Feature,
+    "newsletter-signup-layout",
+    "When ON, we replace the standard article layout with the new signup layout for newsletter sign up pages",
+    owners = Seq(Owner.withEmail("newsletters.dev@guardian.co.uk")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
can we add an expiry date please, just choose something sensible in the future
we use this to remember to remove them when we no longer need